www/js/control/ProfileSettings.jsx

```diff
@@ -219,8 +221,12 @@ const ProfileSettings = () => {
     
     //methods that control the settings
     const uploadLog = function () {
-        UploadHelper.uploadFile("loggerDB")
-    };
+        if(uploadReason != "") {
+            let reason = uploadReason.split('').join('');
```

Comment:
Out of curiosity, what is the reason for `split('').join('')`?
Can you add a comment explaining for others in the future?

Reply:
That was part of the code that I migrated, it was just in an ionic on-the-fly popup before. I'll experiment with removing it and see if I can figure out why. The reason does get used for the filename on the server, so that might be it. I'll investigate and leave a comment.

Reply:
Everything still works if I take that out, and I did not find it in the previous version of the file. It was probably something I was trying when I couldn't get things to upload, and I never took it out. I have removed it since it was unnecessary code.